src/js/angular/similarity/controllers/similarity-list.controller.js

```diff
@@ -253,15 +253,10 @@ function SimilarityCtrl($scope, $interval, toastr, $repositories, $licenseServic
         }
 
         //replace template prefix for PREFIX_INSTANCE in the view mode with actual prefix from the query
-        var array = [];
-        var prefix = '';
-        array = queryTemplate.split(' ');
-        for (let i = 0; i < array.length; i++) {
-            if (array[i].includes(PREFIX_INSTANCE)) {
-                prefix = array[i].substring(0, array[i].indexOf(':'));
-                break;
-            }
-        }
+        let tokens = [];
+        let prefix = '';
+        tokens = queryTemplate.match(/[a-zA-Z0-9-]+:<http:\/\/www.ontotext.com\/graphdb\/similarity\/instance\/>/);
```

Comment:
Are you sure this now actually work? In any case, you should at least verify that the match actually found something, otherwise you will get the equivalent of the NPE in javascript.

Reply:
OK, then just add a check if the `tokens` is defined before trying to access it below to prevent errors.

Reply:
OK. Thanks.